src/book/entities/Account.ts

```diff
@@ -145,73 +145,70 @@ export default class Account extends BaseEntity {
   async save(options?: SaveOptions): Promise<this> {
     const account = await super.save(options);
 
-    const queryClient = Account.getRepository().manager.connection.options.extra?.queryClient;
-    if (queryClient) {
-      updateCache({ queryClient, account: this });
+    if (this.queryClient) {
+      updateCache({ queryClient: this.queryClient, entity: this });
     }
 
     return account;
   }
 
   async remove(options?: SaveOptions): Promise<this> {
-    const queryClient = Account.getRepository().manager.connection.options.extra?.queryClient;
-    if (queryClient) {
-      updateCache({ queryClient, account: this, isDelete: true });
+    if (this.queryClient) {
+      updateCache({ queryClient: this.queryClient, entity: this, isDelete: true });
     }
 
     const account = await super.remove(options);
     return account;
   }
 }
 
+/**
+ * Because accounts have parent/children relations,
+ * whenever an account is updated or deleted we need to update
+ * those relations
+ */
 export async function updateCache(
```

Comment:
this now contains only behavior about updating parent children as the "normal" behavior is in BaseEntity